### ISSUE TITLE
Capability setting whose value is false is still NOT reflected to IE webdriver in protractor 3.3.0

### ISSUE THREAD

**Issue by miyukiw** [angular/protractor#3253]
Since the issue I reported before #3058 seems to be fixed in protractor 3.3.0, I tried to run the same E2E test again with protractor `3.3.0` , and I found `nativeEvents: false` is still NOT reflected to remote webdriver.

My understanding is that none of capability setting whose value is `false` is passed to webdriver with protractor `3.2.1` due to external bug ( #3058 )
And it hasn't been fixed yet with protractor `3.3.0` .

Please confirm.

--

- Node Version: `v4.3.1`
- Protractor Version: `3.3.0`

### protractor configuration file
```
capabilities: {
  'browserName': 'internet explorer',
  'version': '11',
  'selenium-version': '2.53.0',
  'name': 'E2E Tests on IE in WINDOWS',
  'nativeEvents': false,
  'unexpectedAlertBehaviour': 'accept',
  'ignoreProtectedModeSettings': true,
  'disable-popup-blocking': true,
  'enablePersistentHover': true
}
```

### with protractor 3.3.0 

"count": 1,
"name": "E2E Tests on ie in WINDOWS",
"selenium-version": "2.53.0",
"browserName": "internet explorer",
"version": "11",
"unexpectedAlertBehaviour": "accept",
"enablePersistentHover": true,
"disable-popup-blocking": true

### with protractor 3.2.2

"count": 1,
"name": "E2E Tests on ie in WINDOWS",
"selenium-version": "2.53.0",
"browserName": "internet explorer",
"version": "11",
"unexpectedAlertBehaviour": "accept",
"enablePersistentHover": true,
"disable-popup-blocking": true

### with protractor 3.1.1 (works correctly)

"count": 1,
"name": "E2E Tests on ie in WINDOWS",
**"ignoreProtectedModeSettings": false,**
"selenium-version": "2.53.0",
"browserName": "internet explorer",
"version": "11",
**"nativeEvents": false,**
"enablePersistentHover": true,
"unexpectedAlertBehaviour": "accept",
"disable-popup-blocking": true

**Comment by cnishina** [comment]
Hmmm... this should have worked with the updated version of selenium-webdriver. Could you try to see if setting `"nativeEvents": 0` works?

**Comment by miyukiw** [comment]
I tried `0` instead of `false` . Here is the results.

### with 3.3.0
Value `0` as well as `false` wasn't passed to webdriver. Falsy value seems not to be passed :(

```
"count": 1,
"name": "E2E Tests on ie in WINDOWS",
"selenium-version": "2.53.0",
"browserName": "internet explorer",
"version": "11",
"unexpectedAlertBehaviour": "accept",
"enablePersistentHover": 1,
"disable-popup-blocking": 1
```

### with 3.1.1
Value `0` could be passed as well as `false` .
```
"count": 1,
"name": "E2E Tests on ie in WINDOWS",
"ignoreProtectedModeSettings": 0,
"selenium-version": "2.53.0",
"browserName": "internet explorer",
"version": "11",
"nativeEvents": 0,
"enablePersistentHover": 1,
"unexpectedAlertBehaviour": "accept",
"disable-popup-blocking": 1
```
By the way `nativeEvents: 0` doesn't  work the same as `nativeEvents: false` so it doesn't make sense even if it could be passed to webdriver.

**Comment by demsey2** [comment]
I hit the same problem and it looks like in version 3.1.1 webdriver had different implementation for Symbols.serialize

..\node_modules\protractor\node_modules\selenium-webdriver\lib\capabilities.js line: 299
```javascript
let cap = this.get(key);
if (cap) { // condition in version 2.48.2 was (cap !== undefined && cap !== null)
    ret[key] = cap;
}
```
so if "cap" variable is false as we set for "nativeEvents" then is skipped

**Comment by cnishina** [comment]
Investigating this. The fix is with selenium webdriver 2.53.2.  This has been fixed on the master branch.
